### ISSUE TITLE
RemindMe 3.1.03 crashes on start

### ISSUE THREAD

**Issue by brgrvrm** [Stefangansevles/RemindMe#6]
**Describe the bug**
RemindMe 3.1.03 crashes on start

**To Reproduce**
1. Install RemindMe
2. Start RemindMe
3. Voila

**Expected behavior**
Here's the German error log.
```
 - Unknown exception in main.
System.TypeInitializationException: Der Typeninitialisierer für "RemindMe.MUCSettings" hat eine Ausnahme verursacht. ---> System.Runtime.InteropServices.COMException: Die COM-Klassenfactory für die Komponente mit CLSID {6BF52A52-394A-11D3-B153-00C04F79FAA6} konnte aufgrund des folgenden Fehlers nicht abgerufen werden: 80040154 Klasse nicht registriert (Ausnahme von HRESULT: 0x80040154 (REGDB_E_CLASSNOTREG)).
   bei System.RuntimeTypeHandle.CreateInstance(RuntimeType type, Boolean publicOnly, Boolean noCheck, Boolean& canBeCached, RuntimeMethodHandleInternal& ctor, Boolean& bNeedSecurityCheck)
   bei System.RuntimeType.CreateInstanceSlow(Boolean publicOnly, Boolean skipCheckThis, Boolean fillCache, StackCrawlMark& stackMark)
   bei System.RuntimeType.CreateInstanceDefaultCtor(Boolean publicOnly, Boolean skipCheckThis, Boolean fillCache, StackCrawlMark& stackMark)
   bei System.Activator.CreateInstance(Type type, Boolean nonPublic)
   bei System.Activator.CreateInstance(Type type)
   bei RemindMe.MUCSettings..cctor()
   --- Ende der internen Ausnahmestapelüberwachung ---
   bei RemindMe.MUCSettings..ctor()
   bei RemindMe.MaterialForm1.MaterialForm1_Load(Object sender, EventArgs e)
   bei System.Windows.Forms.Form.OnLoad(EventArgs e)
   bei System.Windows.Forms.Form.OnCreateControl()
   bei System.Windows.Forms.Control.CreateControl(Boolean fIgnoreVisible)
   bei System.Windows.Forms.Control.CreateControl()
   bei System.Windows.Forms.Control.WmShowWindow(Message& m)
   bei System.Windows.Forms.Control.WndProc(Message& m)
   bei System.Windows.Forms.ScrollableControl.WndProc(Message& m)
   bei System.Windows.Forms.Form.WmShowWindow(Message& m)
   bei System.Windows.Forms.Form.WndProc(Message& m)
   bei MaterialSkin.Controls.MaterialForm.WndProc(Message& m)
   bei RemindMe.MaterialForm1.WndProc(Message& m)
   bei System.Windows.Forms.Control.ControlNativeWindow.OnMessage(Message& m)
   bei System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)
   bei System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)
```

**Screenshots**
none

**please complete the following information**
 - Version 3.1.03

**Additional context**
Windows 10 Enterprise 1809

**Comment by Stefangansevles** [comment]
Had a look at this, can you try to see if the issue is fixed in 3.1.04?

**Comment by tnira** [comment]
Thank you for providing a useful application.

I installed the application using this [42a9a33](https://github.com/Stefangansevles/RemindMe/blob/42a9a33a2b8d66a4b60bb01f598ab62bce22a921/SetupRemindMe.msi) installer, but the application will not start with the same error code(0x80040154).

```
[2022/01/24 11:09:01] - Unknown exception in main.
System.TypeInitializationException: The type initializer for 'RemindMe.MUCSettings' threw an exception. ---> System.Runtime.InteropServices.COMException: Retrieving the COM class factory for component with CLSID {6BF52A52-394A-11D3-B153-00C04F79FAA6} failed due to the following error: 80040154 Class not registered (Exception from HRESULT: 0x80040154 (REGDB_E_CLASSNOTREG)).
   at System.RuntimeTypeHandle.CreateInstance(RuntimeType type, Boolean publicOnly, Boolean noCheck, Boolean& canBeCached, RuntimeMethodHandleInternal& ctor, Boolean& bNeedSecurityCheck)
   at System.RuntimeType.CreateInstanceSlow(Boolean publicOnly, Boolean skipCheckThis, Boolean fillCache, StackCrawlMark& stackMark)
   at System.RuntimeType.CreateInstanceDefaultCtor(Boolean publicOnly, Boolean skipCheckThis, Boolean fillCache, StackCrawlMark& stackMark)
   at System.Activator.CreateInstance(Type type, Boolean nonPublic)
   at System.Activator.CreateInstance(Type type)
   at RemindMe.MUCSettings..cctor()
   --- End of inner exception stack trace ---
   at RemindMe.MUCSettings..ctor()
   at RemindMe.MaterialForm1.MaterialForm1_Load(Object sender, EventArgs e)
   at System.Windows.Forms.Form.OnLoad(EventArgs e)
   at System.Windows.Forms.Form.OnCreateControl()
   at System.Windows.Forms.Control.CreateControl(Boolean fIgnoreVisible)
   at System.Windows.Forms.Control.CreateControl()
   at System.Windows.Forms.Control.WmShowWindow(Message& m)
   at System.Windows.Forms.Control.WndProc(Message& m)
   at System.Windows.Forms.ScrollableControl.WndProc(Message& m)
   at System.Windows.Forms.Form.WmShowWindow(Message& m)
   at System.Windows.Forms.Form.WndProc(Message& m)
   at MaterialSkin.Controls.MaterialForm.WndProc(Message& m)
   at RemindMe.MaterialForm1.WndProc(Message& m)
   at System.Windows.Forms.Control.ControlNativeWindow.OnMessage(Message& m)
   at System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)
```

**environment**
Windows 11 Pro 21H2
